prepared.go

```diff
@@ -0,0 +1,20 @@
+package tarantool
+
+type PreparedID uint64
+
+// Prepared is a type for handling prepared statements
+//
+// Since 1.7.0
+type Prepared struct {
+	StatementID PreparedID
+	MetaData    []ColumnMetaData
+	ParamCount  uint64
+	Conn        *Connection
+}
+
+// NewPreparedFromResponse constructs a Prepared object.
+func NewPreparedFromResponse(conn *Connection, resp *Response) *Prepared {
```

Comment:
It would be greate to return an error here insead of runtime error if response is not valid (we need a test for the case too):
```suggestion
func NewPreparedFromResponse(conn *Connection, resp *Response) *Prepared, error {
```

Reply:
Fixed.